### ISSUE TITLE
Disable token revocation property is not working in APIM 4.2.0

### ISSUE THREAD

**Issue by SupunRandunu** [wso2/api-manager#2146]
### Description

The following disable token revocation property is not working in APIM 4.2.0

```
[oauth.jwt.renew_token_without_revoking_existing]
enable = true
```

### Steps to Reproduce

1. Add the property
```
   [oauth.jwt.renew_token_without_revoking_existing]
   enable = true
```
4. Start the Server
5. Generate an access token and invoke API
6. Generate another token before the initial one is expired
7. Reinvoke with the initial token and observe that is not valid anymore.

### Affected Component

APIM

### Version

4.2.0

**Comment by BLasan** [comment]
Hi,

Reproduced the issue. Will start porting the fix on 4.2.0

Thanks,
Benura

**Comment by YasasRangika** [comment]
Public fixes:
https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2053
https://github.com/wso2/carbon-identity-framework/pull/4535
have been merged into the master branches.
